catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -90,7 +90,7 @@ def create_aperture(self):
                                                           flattoflat=self.flat_to_flat,
                                                           gap=self.gap,
                                                           segmentlist=self._segment_list,
-                                                          rotation=-90)
+                                                          rotation=0)
```

Comment:
@kjbrooks do we know the "real world" orientation of the JOST IrisAO? Does JOST have a single corner segment in the upwards direction like the spare HiCAT IrisAO here, or one of the flat sides with segments in a straight line? I am not sure what kind of rotations the camera, Sharpcap... introduce, so we might need to check this in the lab right in front of the camera next time somebody is in. We can probably do this bye applying a global defocus on the IrisAO (either by commanding the mirror or by translating the camera).
In any case, we might want to consider to pull this parameter out so that it is changeable. Also, it's only used for the command display, so this is not crucial for the controls, but it's easier to work with the commands if they're oriented correctly.

Reply:
For JOST, I think it's flat side up. I did a lot of testing and -90 is correct for JOST so I recommend that you make this a variable instead of making a hard coded change since that will mess up JOST. I am all for looking more into this, but if we want it to work on both of them right now, we will need to make this a variable.